land_use/urls.py

```diff
@@ -0,0 +1,37 @@
+from django.urls import path
+
+from land_use import views
+
+
+urlpatterns = [
+    path(
+        '',
+        views.LandUseView.as_view(),
+        name='mapping'
+    ),
+    path(
+        'detail/<int:id>/',
+        views.LandUseDetailView.as_view(),
+        name='mapped-detail'
+    ),
+    path(
+        'years/',
```

Comment:
It might filter an funai indigenous lands, right?

Reply:
Is already filtering.